src/backend/distributed/connection/placement_connection.c

```diff
@@ -32,45 +32,6 @@
 #include "utils/memutils.h"
 
 
-/*
```

Comment:
This structure keeps an internal state, and better suited to stay in this file. Can we avoid making this struct public (e.g., not move to the header)? Maybe by accessing the relevant fields via a function?

Reply:
If we do the suggestions mentioned here: https://github.com/citusdata/citus/pull/4587/files#r565098083, we won't need to use anything from this struct I think, hence it can go back without introducing any function

Reply:
The function already exists: https://github.com/citusdata/citus/blob/master/src/backend/distributed/connection/placement_connection.c#L828-L861

Reply:
Cool, we can directly use that function instead of adding a new field to connection struct then